packages/cf-component-radio/src/RadioTheme.js

```diff
@@ -0,0 +1,15 @@
+export default baseTheme => ({
+  cursor: 'pointer',
+  display: 'block',
+  minHeight: '1em',
+  paddingLeft: '2em',
+  marginTop: 'initial',
+
+  '&:first-child': {
+    marginTop: 'initial'
+  },
+
+  '&:hover input': {
+    borderColor: '#256298'
```

Comment:
This color: `#256298`.

Marine:       `#2F7BBF`.
Rain:            `#408BC9`.

Is either one of those close enough to pick?